Microsoft.Toolkit.Wpf.UI.Controls/MapControl/MapControl.cs

```diff
@@ -156,11 +152,6 @@ private void RelocateChildToUwpControl(WindowsXamlHostBase obj)
         public static DependencyProperty LocationProperty { get; } = DependencyProperty.Register(nameof(Center), typeof(Geopoint), typeof(MapControl));
         */
 
-        /// <summary>
-        /// Gets <see cref="windows.UI.Xaml.Controls.Maps.MapControl.MapElementsProperty"/>
-        /// </summary>
-        public static DependencyProperty MapElementsProperty { get; } = DependencyProperty.Register(nameof(MapElements), typeof(System.Collections.Generic.IList<MapElement>), typeof(MapControl));
```

Comment:
Regarding the removal of Dependency Properties, I assume that means data binding on those will be not possible. Am I missing something?